skimage/util/arraycrop.py

```diff
@@ -4,6 +4,7 @@
 """
 
 import numpy as np
+from numpy.lib.arraypad import _as_pairs
```

Comment:
didn't we learn our lesson from importing underscore functions?

Reply:
We did but the code does that already, so this PR does not make things worse. I would raise an issue to vendor this code, but generally I'm of the opinion that PRs aren't required to "incidentally" fix things that were not the primary point of the PR.

Reply:
Opened in #5028